apiv2/database/models/tomogram.py

```diff
@@ -72,9 +72,9 @@ class Tomogram(Base):
         back_populates="tomograms",
     )
     name: Mapped[str] = mapped_column(String, nullable=True)
-    size_x: Mapped[float] = mapped_column(Float, nullable=False)
-    size_y: Mapped[float] = mapped_column(Float, nullable=False)
-    size_z: Mapped[float] = mapped_column(Float, nullable=False)
```

Comment:
We should add `is_visualization_default`, `publications` and `related_database_entries` fields to this table as well.

Reply:
Resolving it here, as I added this [comment to the other PR.](https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/280#discussion_r1774575521)